query_execution.go

```diff
@@ -102,11 +102,11 @@ func (q *queryExecution) Execute(queryPlan *QueryPlan) ([]executionResult, gqler
 
 func (q *queryExecution) executeRootStep(step *QueryPlanStep) error {
 	var document string
-	if step.ParentType == "Query" {
-		document = "query " + formatSelectionSet(q.ctx, q.schema, step.SelectionSet)
-	} else if step.ParentType == "Mutation" {
-		document = "mutation " + formatSelectionSet(q.ctx, q.schema, step.SelectionSet)
-	} else {
+
+	switch operationType := step.ParentType; operationType {
+	case "Query", "Mutation":
```

Comment:
`queryObjectName` and `mutationObjectName` are defined in `schema.go` in case you want to use consts for those values